### PR TITLE
Styling tweaks + Deselect zone when going back to map on mobile

### PR DESCRIPTION
**Pull request by madsnedergaard**
## Issue

https://linear.app/electricitymaps/issue/ELE-1570/zone-should-be-deselected-on-mobile-when-navigating-back-to-the-map

## Description

This resets the state of the zone when clicking on the "back to map" arrow in zone panel on mobile.
Also made some minor tweaks so we have almost everything from Rasmus fixed! :)

### Preview

https://user-images.githubusercontent.com/3296643/211854024-35a82f9f-c426-4160-b57f-3abad27665e0.mp4

### CODE REVIEW

File: web/src/components/CircularGauge.tsx [hunk 1]
```diff
@@ -25,12 +25,12 @@ export function CircularGauge({ ratio, name }: CircularGaugeProps) {
...
           dataKey="value"
           data={[{ value: 100 }]}
-          className="fill-gray-200 dark:fill-gray-600/50"
+          className="fill-gray-200/60 dark:fill-gray-600/50"
           isAnimationActive={false}
           strokeWidth={0}
         >
           <Label
-            className="select-none fill-gray-900 text-md font-bold dark:fill-gray-300"
+            className="select-none bg-red-500 fill-gray-900 font-bold dark:fill-gray-300"
```
Comment (madsnedergaard) [inline web/src/components/CircularGauge.tsx:33]:
```suggestion
            className="select-none fill-gray-900 font-bold dark:fill-gray-300"
```

**Comment by tonypls** [comment]
Not sure when it happened but the left panel carbon intensity square and gauge font is much larger than the tooltip
image

**Comment by madsnedergaard** [comment]
> Not sure when it happened but the left panel carbon intensity square and gauge font is much larger than the tooltip image

Rasmus pointed out in Figma that it should be larger (like the current app), but I changed it in the components, so don't know why the tooltip is not using larger text size too 🤔  I'll investigate

**Review by tonypls — APPROVED** [review]
LGTM

**Pull request merged**